joomla/joomla-cms
# [4.0] Implementation of feature 'filter menu items by component'

Pull Request for this Issue [here](https://github.com/joomla/joomla-cms/pull/21358 "")


### Summary of Changes
We built a filter for menu items by components so that we can have now the possibility to filter items or subitems that belongs to a special component.


### Testing Instructions
After creating new menu items (of same and different component types) in differents menus you click on the dropdown menu "Table options" and select the menu " Select Component Menu Item " and then you can choose which items of which component you want to display.
You should test on the site backend (not the administrator backend)


### Expected result
Items of the components choosed must be showed.


### Actual result
Items of the components choosed are showed.



### Documentation Changes Required
